src/components/application_manager/rpc_plugins/rc_rpc_plugin/src/commands/mobile/set_interior_vehicle_data_request.cc

```diff
@@ -59,10 +59,11 @@ std::vector<std::string> GetModuleReadOnlyParams(
     module_ro_params.push_back(kState);
     module_ro_params.push_back(kSisData);
   }
+
   return module_ro_params;
 }
 
-const std::map<std::string, std::string> GetModuleTypeToDataMapping() {
+std::map<std::string, std::string> GetModuleTypeToDataMapping() {
   std::map<std::string, std::string> mapping = {
       {enums_value::kRadio, message_params::kRadioControlData},
       {enums_value::kClimate, message_params::kClimateControlData},
```

Comment:
Looks like SeatControlData is missing here

Reply:
@yang1070, it's rc light PR. SeatControlData will be in PR related to rc seat control